makeFastInput.py

```diff
@@ -64,11 +64,11 @@ def msplit(line):
     if skip: continue
     # parse filename: model, mMother-X, mLSP-Y, MC####, fast.root
     #             or: model, mZprime-X, mDark-Y, rinv-Z, alpha-W, MC####, scan.root
-    splitnum = int(options.params) + 2
+    splitnum = int(options.params) + (1 if options.full else 2)

```

Comment:
this will break the SVJ functionality
instead, how about adding another argument like -x, --extra-params that defaults to 2, but you can set to 1 for the fullsim SUSY case

Reply:
Changed